dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/XmlTool.java

```diff
@@ -38,9 +42,7 @@
 import com.dotcms.repackage.org.dom4j.io.SAXReader;
 
 import com.dotmarketing.util.Logger;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.util.Logger'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)